doc/fuzzing.md

```diff
@@ -153,13 +153,17 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
-Full configuration step that was tested on macOS with `brew` installed `llvm`:
+Currently the default linker on macOS `ld` does not work with recent versions of `llvm` so you will need to install llvm's linker `lld` and use it with the CMAKE_EXE_LINKER_FLAG `-fuse-ld=lld`.
+
+Full configuration step for macOS:
 
 ```sh
+$ brew install llvm lld
 $ cmake --preset=libfuzzer \
    -DCMAKE_C_COMPILER="$(brew --prefix llvm)/bin/clang" \
    -DCMAKE_CXX_COMPILER="$(brew --prefix llvm)/bin/clang++" \
-   -DAPPEND_LDFLAGS=-Wl,-no_warn_duplicate_libraries
+   -DAPPEND_LDFLAGS=-Wl,-no_warn_duplicate_libraries \
```

Comment:
`no_warn_duplicate_libraries` isn't an lld option, so this can be dropped.

Reply:
Thanks, dropped.

Reply:
Nice, finally we can get rid of that (tested, seems to be working without the warning, good find)!
Should we anything to https://github.com/llvm/llvm-project/issues/102630?